### PR TITLE
[KIWI-1230]- F2F | Welsh Translation | Additional Copy Changes in F2F Flow (Kiwi Team)

### PR DESCRIPTION
**Pull request by GDainton**
## Proposed changes

Updating F2F journey with copy changes for parts not covered in the current translation

### What changed

ID choice hints, country of Issue for EU driving licence, EEA ID card and Non-UK passport dropdown menu, address check and yes/no expiry date selection translated into Welsh. Check my answers page displays all catagories and choices in Welsh

### Why did it change

Changes were made so that Welsh speaking users can navigate the F2F journey in its entirety in support of an issued VC

### Issue tracking

- [KIWI-1230](https://govukverify.atlassian.net/browse/KIWI-1230)

## Checklists
Screenshot 2023-11-02 at 14 32 09
Screenshot 2023-11-02 at 14 32 21
Screenshot 2023-11-02 at 14 32 36
Screenshot 2023-11-02 at 14 32 54

[KIWI-1230]: https://govukverify.atlassian.net/browse/KIWI-1230?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/f2f/steps.js [hunk 1]
```diff
@@ -217,12 +217,12 @@ module.exports = {
     next: [
       {
         field: "idHasExpiryDate",
-        value: APP.HAS_EXPIRY_DATE.YES,
```
Comment (Caitlin-cooling) [inline src/app/f2f/steps.js:220]: Why aren't we using these constants any more?

File: src/app/f2f/controllers/checkDetails.js [hunk 2]
```diff
@@ -99,36 +98,35 @@ class CheckDetailsController extends DateController {
...
           break;
         }
         case APP.PHOTO_ID_OPTIONS.EEA_IDENTITY_CARD: {
           idHasExpiryDate = req.form.values.idHasExpiryDate
           expiryDate = req.form.values.eeaIdCardExpiryDate;
-          country = req.form.values.eeaIdentityCardCountrySelector;
           address = req.form.values.eeaIdentityCardAddressCheck;
+          countryCode = req.form.values.eeaIdentityCardCountrySelector;
+          req.sessionModel.set("countryCode", countryCode);
+          req.sessionModel.set("country", res.locals.translate(`countries.${countryCode}`))
```
Comment (Caitlin-cooling) [inline src/app/f2f/controllers/checkDetails.js:121]: Do we need to set this is 3 places or can we just do `locals.countryTranslatedString = res.locals.translate(countries.${countryCode})`?

Comment (GDainton) [reply src/app/f2f/controllers/checkDetails.js:121]: The countryCode can't be set outside of those cases as it then overrides the UK ID options which gives rise to an undefined value for countryCode for UK passport, BRP and UK driving licence. The country and countryCode need to be set on the sessionModel for the purposes of testing, and the countryTranslatedString is just the display value for the check details screen

Comment (GDainton) [reply src/app/f2f/controllers/checkDetails.js]: I've removed countryCode as a variable as it wasn't needed, but have used the sessionModel variable country to set the countryTranslatedString as it's shorter - the reason for the country sessionModel variable is for the existing test expectations but apart from that it isn't used for anything

**Review by Caitlin-cooling — APPROVED** [review]

**Pull request merged**